tests/test_admix.py

```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import warnings
 """Tests for `admix` package."""
 
+# This test seems desperatly out of date
```

Comment:
Again i would just remove if they weren't working -- i never used them at least

Reply:
Sure sounds good, just wanted to have you on board before deleting other peoples work